docs/data/indexers/build-your-own/galexie/admin_guide/configuring.mdx

```diff
@@ -7,41 +7,47 @@ sidebar_position: 20
 
 ## Steps to Configure Galexie
 
-### 1. Copy the Sample Configuration
+### Kubernetes
```

Comment:
I don't think need to mention k8s on this page since the page's purpose is mostly runtime environment agnostic and more focused on just config data to network and bucket storage.